.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[target.aarch64-unknown-linux-gnu]
+linker = "aarch64-linux-gnu-gcc"
+[target.i686-unknown-linux-gnu]
+linker = "i686-linux-gnu-gcc"
```

Comment:
Sorry, can I ask why these are necessary? Shouldn't cross-compiling always find an appropriate cross-linker? (And if you're *not* cross-compiling, shouldn't you use the default linker?)

Reply:
Without this, the build breaks because it will use the x64 linker. Also see https://wiki.pine64.org/index.php?title=Cross-compiling&mobileaction=toggle_view_desktop#cargo. This only applies to Linux, on Windows and macOS this is not needed.

Your comment about not cross-compiling makes sense, so what I could do is add some logic to `bin/build-electron` that if the toolchain is `x86_64-unknown-linux` (the logic is already there), and the target is `aarch64` or `i686`, it uses said linkers. What do you think about that?

Reply:
I guess my hesitancy comes from writing down specific linkers *anywhere* in a user-invoked build process. If I use gold as my cross-linker rather than gcc, the build's going to fail here. What do you think about moving the setting to an [environment variable](https://doc.rust-lang.org/cargo/reference/config.html#targettriplelinker) in the GitHub workflow, rather than having it be a default in the build scripts?